test/compress/issue-208.js

```diff
@@ -27,3 +27,28 @@ do_update_rhs: {
         MY_DEBUG += 0;
     }
 }
+
+mixed: {
+    options = {
+        global_defs: { DEBUG: 0 }
+    }
+    input: {
```

Comment:
Having `const DEBUG = true;` at the top of the file and overriding it with `--define DEBUG=false` would be a common use case. It would work well with `-c toplevel`.